scripts/netem.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+#
+# Start/Stop network emulation
+#
+set -e
+
+[[ $(uname) == Linux ]] || exit 0
+
+cd "$(dirname "$0")"
+
+sudo=
+if sudo true; then
+  sudo="sudo -n"
+fi
+
```

Comment:
might want to slip a `set -x` in here to produce output that's helpful for debugging when something goes wrong